toc/rfc/rfc-draft-use-github-packages-container-registry-instead-of-dockerhub.md

```diff
@@ -0,0 +1,37 @@
+# Meta
+[meta]: #meta
+- Name: Distribute Public Community Images using Github Packages Container Registry instead of DockerHub
+- Start Date: 2024-04-18
+- Author(s): @tcdowney
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (fill in with PR link after you submit it)
+
+
+## Summary
+
+Move away from DockerHub and distribute official Cloud Foundry container images using the [Github Packages](https://github.com/features/packages) [Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). This RFC is primarily concerned with public container images that are published for the Cloud Foundry community.
+
+## Terminology
+
+* **Community Image -** A public container image that is intended to be consumed by members of the Cloud Foundry community.  Some examples are the `cloudfoundry/cli` and `cloudfoundry/cflinuxfs4` images that are published by the CLI and Buildpacks teams.
+* **Team Image -** A container image that a team publishes for its own internal use (e.g. for use in CI). These images may or may not be public.
+
+## Problem
+
+Currently some working groups distribute container images for consumption by the Cloud Foundry community through the [CloudFoundry DockerHub org](https://hub.docker.com/repositories/cloudfoundry).
+
+Managing DockerHub accounts adds additional overhead to the release processes for these teams. By switching to using Github Packages, teams can use their existing Github accounts to publish images.
+
+It is also unclear what the long-term costs of using DockerHub will be. I believe that the `cloudfoundry` org is currently covered by the [Docker-sponsored Open Source program](https://docs.docker.com/trusted-content/dsos-program/), but [Github Package usage is free for public packages](https://docs.github.com/en/billing/managing-billing-for-github-packages/about-billing-for-github-packages#about-billing-for-github-packages).
```

Comment:
@christopherclark my knowledge on this could be stale, so please let me know if we're still covered by their DSOS program.

Reply:
cloudfoundry is NOT covered by the DSOS program, but we pay for that org (about $500/month). Paketo buildpacks _is_ covered by the DSOS program. You have to re-apply annually, for each DH organization, and TBH the criteria for this program is opaque, and they aren't very communicative about how it works.